modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -2,17 +2,17 @@
 
 == Broker-wide throughput limits
 
 The network bandwidth and disk utilization of brokers may be overloaded by clients that produce or consume throughput without limits. To prevent resource overloading caused by unconstrained throughput and to configure back pressure, Redpanda provides runtime-configurable properties that limit and balance throughput of Kafka API traffic.
 
 To manage the volume of traffic going through a broker, Redpanda implements throughput quotas on the ingress and egress sides of every broker. The throughput quota accounts for all Kafka API traffic going in or out of a broker, with the value of quota representing the allowed rate of data passing through in one direction. When a connection is in breach of the quota, the throttler advises the client about the delay (throttle time) that would bring the rate back to the allowed level, and it implements that delay before handling Kafka API requests. To control the quotas, Redpanda provides configurable rate limits for total ingress and egress traffic through a broker.
 
-With Redpanda's xref:get-started:architecture.adoc#thread-per-core-model[thread-per-core model], the Kafka API traffic to and from a client connection is processed by a single core (shard). In order to manage throughput quotas efficiently, broker quotas are distributed between shards, and each per-shard quota is in turn shared by all connections served by the shard. Splitting broker quota optimally between shards is done behind the scenes by the quota balancer component.
+== Advanced throughput limit tuning
 
-To distribute the broker throughput quota, the balancer periodically monitors the throughput rate of a broker's shards, and it distributes more quota to the shards that can make better use of it than the others. Each shard has a minimum throughput quota value, which is configurable both as a percentage of the default quota and as an absolute rate limit.
+The throttling mechanism separates ingress and egress quotas and applies them across all CPU cores. When one of the throttling quotas is reached, Redpanda adds a throttling delay, `throttle_time_ms`, to subsequent responses. If the client still fails to back off, then the next read operation is delayed. Redpanda attempts to set the throttling delay to an amount that will put the cluster back below the throttling threshold. This delay will not exceed the value set in `max_kafka_throttle_delay_ms`. Administrators may also use the tunable property  xref:reference:tunable-properties.adoc#kafka_throughput_replenish_threshold[`kafka_throughput_replenish_threshold`] to adjust the rate that Redpanda eases throttling once it is engaged.
```

Comment:
If you can actually count something (like clusters, or seconds, etc), use _number_. If it is something hard to quantify, like the number of grains in a cup of sugar, use _amount_. Seems like you are using amount here when you should be using number...

Reply:
Switchied to 'number'.